paperwork-backend/paperwork_backend/shell.py

```diff
@@ -633,6 +653,17 @@ def cmd_import(*args):
         docid = args[idx + 1]
         args.pop(idx)
         args.pop(idx)
+    if "--name" in args:
```

Comment:
Please update the documentation of the command: It's the docstring of the function `cmd_import`.